src/Illuminate/Collections/Enumerable.php

```diff
@@ -499,6 +499,14 @@ public function has($key);
      */
     public function implode($value, $glue = null);
 
+    /**
+     * Explode collection's values base on the separators
+     *
+     * @param string|array<string>  $separators
+     * @return static
+     */
+    public function explode($separators);
```

Comment:
adding to an interface is a major breaking change.

Reply:
Oh true... Did not think of it thanks !